.github/workflows/linters.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - uses: hadolint/hadolint-action@v3.0.0
         with:
           recursive: true
-          ignore: DL3041
```

Comment:
Is this change related?

Reply:
Yes, since we changed to alpine, need to update ignore rule as well